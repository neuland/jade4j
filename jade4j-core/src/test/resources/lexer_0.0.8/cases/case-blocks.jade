html
  body
    - var friends = 1
    case friends
      when 0
        p you have no friends
      when 1
        p you have a friend
      default
        p you have #{friends} friends