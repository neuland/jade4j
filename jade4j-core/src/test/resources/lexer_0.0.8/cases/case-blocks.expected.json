{"type":"tag","line":1,"val":"html","selfClosing":false}
{"type":"indent","line":2,"val":2}
{"type":"tag","line":2,"val":"body","selfClosing":false}
{"type":"indent","line":3,"val":4}
{"type":"code","line":3,"val":"var friends = 1","escape":false,"buffer":false}
{"type":"newline","line":4}
{"type":"case","line":4,"val":"friends"}
{"type":"indent","line":5,"val":6}
{"type":"when","line":5,"val":"0"}
{"type":"indent","line":6,"val":8}
{"type":"tag","line":6,"val":"p","selfClosing":false}
{"type":"text","line":6,"val":"you have no friends"}
{"type":"outdent","line":7}
{"type":"when","line":7,"val":"1"}
{"type":"indent","line":8,"val":8}
{"type":"tag","line":8,"val":"p","selfClosing":false}
{"type":"text","line":8,"val":"you have a friend"}
{"type":"outdent","line":9}
{"type":"default","line":9}
{"type":"indent","line":10,"val":8}
{"type":"tag","line":10,"val":"p","selfClosing":false}
{"type":"text","line":10,"val":"you have #{friends} friends"}
{"type":"outdent","line":10}
{"type":"outdent","line":10}
{"type":"outdent","line":10}
{"type":"outdent","line":10}
{"type":"eos","line":10}